### ISSUE TITLE
ui-combo dropdown menu blocked by ui-panel

### ISSUE THREAD

**Issue by emcga** [adarshpastakia/aurelia-ui-framework#42]
@adarshpastakia  When I place a ui-combo element at the bottom of a ui-panel it's dropdown menu disappears into the panel when activated. It would be nice to have the dropdown menu set to be the topmost element on the page when it is active. I tried to adjust its z-index value without success. Thank you for your time.

**Comment by emcga** [comment]
@adarshpastakia Here's a screenshot of the combobox. The last ui-combo at the bottom of the first ui-panel.

![screen shot 2016-05-06 at 5 30 24 pm](https://cloud.githubusercontent.com/assets/13136877/15086666/55cb76da-13b0-11e6-9807-f9534c025d23.png)

**Comment by emcga** [comment]
Here's my HTML for the panel


```html
 <!-- Inventory item information-->
        <ui-panel>
          <ui-header secondary collapse="true">item info</ui-header>

          <ui-body>
              <ui-form class="ui-pad-v ui-pad-right-30" validate.bind="validation" submit.trigger="onSubmit()">
                <ui-row>
                  <ui-column>
                    <!-- Binds to the manufacturer of the inventory item-->
                    <ui-input name required clear value.bind="inventoryItem.manufacturer">Manufacturer</ui-input>
                    <!-- Binds to the model of the inventory item-->
                    <ui-input name required clear value.bind="inventoryItem.model">Model</ui-input>
                    <!-- Binds to the serial number of the inventory item-->
                    <ui-input name clear value.bind="inventoryItem.serial">Serial</ui-input>
                    <!-- Binds to the classification of the inventory item-->
                    <ui-combo name required clear label="Classification" value.bind="inventoryItem.classification">Classification</ui-combo>
                    <!-- Binds to the lot of the inventory item-->
                    <ui-combo name required clear label="Lot" addon-class="ui-flag" value.bind="inventoryItem.lot">Lot</ui-combo>
                  </ui-column>
                </ui-row>
              </ui-form>
          </ui-body>
        </ui-panel>

**Comment by emcga** [comment]
I just noticed that other people are having the same issue. Sorry for opening a duplicate issue.

**Comment by adarshpastakia** [comment]
@emcga I have updated a fix.
if the combo is within a scrollable element, the list will open upwards if there no space

**Comment by emcga** [comment]
Thank you for your time, it works great now!
